.gitignore

```diff
@@ -33,5 +33,6 @@ trento-*.tar
 # In case you use Node.js/npm, you want to ignore these.
 npm-debug.log
 /assets/node_modules/
+/assets/.gitignore
```

Comment:
I don't get this. Do we need other `.gitignore` file? We already have one in the root folder.
Maybe you meant there `/assets/.env` ?

Reply:
What I was thinking while committing this?